python/tk_multi_workfiles/entity_tree/entity_tree_form.py

```diff
@@ -113,6 +126,20 @@ def __init__(self, entity_model, search_label, allow_task_creation, parent):
         if selection_model:
             selection_model.selectionChanged.connect(self._on_selection_changed)
 
+    def _model_about_to_reset(self):
+        # Catch the currently selected item and convert it to dictionary form
+        # so we can pick it back after the model is reset.
+        if self._current_item_ref:
```

Comment:
I'm having trouble finding where `_current_item_ref` is being set to something other than None. If I'm just blind please ignore this!